nodes/src/nodes/ml_sklearn/Instance.py

```
@@ -0,0 +1,15 @@
+    '''ML Sklearn Instance Node'''
+
+    def __init__(self, IGlobal):
+        self.IGlobal = IGlobal
+
+    def process(self, text):
+        try:
+            preprocessor = self.IGlobal.preprocessor
+            if preprocessor:
+                return preprocessor.process(text)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)